charts/agent/templates/configmap.yaml

```diff
@@ -13,20 +13,10 @@ data:
 {{- if $clusterName }}
     k8s_cluster_name: {{ $clusterName }}
 {{- end }}
-{{- if or .Values.secure.enabled .Values.auditLog.enabled }}
-    security:
-  {{- if .Values.auditLog.enabled }}
-      k8s_audit_server_url: {{ .Values.auditLog.auditServerUrl }}
-      k8s_audit_server_port: {{ .Values.auditLog.auditServerPort }}
-  {{- end }}
-  {{- if .Values.secure.enabled }}
-      enabled: true
-    commandlines_capture:
-      enabled: true
-    memdump:
-      enabled: true
-  {{- end }}
+{{- if and (not .Values.secure.enabled) .Values.auditLog.enabled }}
+  {{- fail "auditLog.enabled cannot be true when secure.enabled is false" }}
```

Comment:
tiny suggestion: might make sense to do it from within the agent.secureFeatures?

Reply:
Sure, I can do this. I think I tried that before and ended up with weird blank lines that I couldn't solve with adding `-`'s, but I agree with you it would look cleaner in the template. Let me try again. 💪🏻

Reply:
💪

Reply:
Done!